fvtest/tril/tril/ast.hpp

```diff
@@ -28,6 +28,25 @@
 #include <assert.h>
 #include <cstring>
 
+#if __cplusplus <= 199711L
+const class nullptr_t {
```

Comment:
This is kind of silly but nullptr was only added as recent as C++11. And just using NULL seems like a very crude solution (maybe I'm missing something here).
Open to suggestions as to whether there's a more elegant workaround.

Reply:
I'll admit to my gut not having a good feeling about what the 'right' answer here is, but, I would suggest this might be best put in another header; I don't think it makes sense here.

Reply:
I would really prefer if `NULL` was used.  Whenever possible we should try to be consistent throughout the entire codebase.